### ISSUE TITLE
Add data type back to field_metadata

### ISSUE THREAD

**Issue by peterdesmet** [inbo/etn#155]
The table `app.field_metadata` seems to replace the previous `vliz.field_definitions`. Field names have been renamed but I could find the columns we originally had. Except for `datatype` (character varying, text, double precision, etc.). @aubrivliz can that be added again?

Columns as we had it previously:

<img width="879" alt="Screenshot 2021-05-17 at 10 46 57" src="https://user-images.githubusercontent.com/600993/118460839-5e4cba80-b6fd-11eb-8376-e2e3880670f6.png">

**Comment by aubrivliz** [comment]
@peterdesmet is this still relevant or can this be closed?

**Comment by peterdesmet** [comment]
I think it is still useful to include `datatype` in `app.field_metadata`. Do you think that is possible?

**Comment by aubrivliz** [comment]
All the information is provided in `app.field_metadata.field_units_format_description`. Do you have enouph information with that field? 

Example: `format: integer; as in: 1003`

**Comment by peterdesmet** [comment]
Yes, as the content is for humans (not machines) that should be sufficient.
